src/command_modules/azure-cli-network/azure/cli/command_modules/network/tests/latest/test_network_commands.py

```diff
@@ -2266,21 +2266,35 @@ def test_network_watcher_vm(self, resource_group, resource_group_location):
         self.cmd('network watcher show-security-group-view -g {rg} --vm {vm}')
         self.cmd('network watcher show-next-hop -g {rg} --vm {vm} --source-ip 123.4.5.6 --dest-ip 10.0.0.6')
 
-    @ResourceGroupPreparer(name_prefix='cli_test_nw_flow_log', location='westcentralus')
-    @StorageAccountPreparer(name_prefix='clitestnw', location='westcentralus')
+    @ResourceGroupPreparer(name_prefix='cli_test_nw_flow_log', location='eastus')
+    @StorageAccountPreparer(name_prefix='clitestnw', location='eastus')
     def test_network_watcher_flow_log(self, resource_group, resource_group_location, storage_account):
 
         self.kwargs.update({
             'loc': resource_group_location,
             'nsg': 'nsg1',
-            'sa': storage_account
+            'sa': storage_account,
+            'ws': self.create_random_name('testws', 20),
+            'la_prop_path': os.path.join(TEST_DIR, 'loganalytics.json')
         })
 
         self.cmd('network nsg create -g {rg} -n {nsg}')
         self.cmd('network watcher flow-log configure -g {rg} --nsg {nsg} --enabled --retention 5 --storage-account {sa}')
         self.cmd('network watcher flow-log configure -g {rg} --nsg {nsg} --retention 0')
         self.cmd('network watcher flow-log show -g {rg} --nsg {nsg}')
 
+        # test traffic-analytics features
+        self.cmd('resource create -g {rg} -n {ws} --resource-type Microsoft.OperationalInsights/workspaces -p @"{la_prop_path}"')
+        self.cmd('network watcher flow-log configure -g {rg} --nsg {nsg} --workspace {ws}', checks=[
```

Comment:
Should we also check here that 'flowAnalyticsConfiguration.networkWatcherFlowAnalyticsConfiguration.enabled' is True?

Reply:
Couldn't hurt